cli/src/main.rs

```diff
@@ -118,6 +118,9 @@ struct Tweaks {
     /// Enables tiling of the output image
     #[structopt(long = "tiling")]
     enable_tiling: bool,
+    /// Flips and rotates traning images
+    #[structopt(long = "flips-and-rotates")]
+    enable_flips_and_rotates: bool,
```

Comment:
I would consider this "flips and rotates mode" different enough to warrant being in its own subcommand, as I don't really see it making sense to also add inpainting and tiling, as you kind of get a tiling effect anyway (in a way).

Reply:
>I would consider this "flips and rotates mode" different enough to warrant being in its own subcommand

Just to double check (since you mention subcommand), you mean something like?
```rust
#[derive(StructOpt)]
enum Subcommand {
    /// Transfers the style from an example onto a target guide
    #[structopt(name = "transfer-style")]
    TransferStyle(TransferStyle),
    /// Generates a new image from 1 or more examples
    #[structopt(name = "generate")]
    Generate(Generate),
    /// Generates a new image from 1 or more flipped and rotated examples
    #[structopt(name = "flips-and-rotates")]
    FlipsAndRotates(FlipsAndRotates),
}
```

or do you mean something else, like moving `enable_flips_and_rotates` to `struct Generate`?

Reply:
Yes, like that. Not sure about the naming though, but it can be changed later if anyone comes up with a clearer and/or shorter name. :slightly_smiling_face:

Reply:
Addressed in 5fdab549e8d229e25d669c3896bd356edb7bc5f1